test/utils/transclude-directive.spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Directive:  pfTransclude', function () {
+  var $scope;
+  var $compile;
+  var outerId;
+  var directiveId;
+  var childId;
+  var parentId;
```

Comment:
Its a preference thing and I have a strong preference for not declaring multiple variables on a single line.